docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,189 @@ Authorization
 #############
 
 When developing Daml applications using SDK tools,
-your local setup will most likely not verify any authorization -
+your local setup will most likely not perform any ledger API request authorization --
 by default, any valid ledger API request will be accepted by the sandbox.
 
-To run your application against a :doc:`deployed ledger </deploy/index>`, you will need to add authorization.
+This is not the case for participant nodes of :doc:`deployed ledgers </deploy/index>`.
+They check for every ledger API request whether the request contains an access token that is valid and sufficient to authorize the request.
+You thus need to add support for authorization using access token to your application to run it against a deployed ledger.
 
 Introduction
 ************
 
-The :doc:`Ledger API </app-dev/ledger-api>` is used to request changes to the ledger (e.g., "*Alice
```

Comment:
"ledger API" (capitalization)

Reply:
https://docs.daml.com/concepts/glossary.html#ledger-api is the current definition that I've used. The capitalization of "Ledger API" makes sense to as it is a standalone concept.

If you'd like to change it, then I'd suggest to do it as a search-and-replace in a separate PR to ensure consistency.

Reply:
No worries, if there's a conceptual reason for it to be capitalized here that works.